ocsf_validator/validators.py

```diff
@@ -223,3 +228,55 @@ def validate(reader: Reader, file: str):
             found[t][name].append(file)
 
     reader.apply(validate, AnyMatcher([ObjectMatcher(), EventMatcher()]))
+
+
+def _default_get_registry(reader: Reader, base_uri: str) -> referencing.Registry:
+    registry = referencing.Registry()
+    for schema_file_path in (reader.base_path / "metaschema").rglob("*.schema.json"):
+        with open(schema_file_path, 'r') as file:
+            schema = json.load(file)
+            resource = referencing.Resource.from_contents(schema)
+            registry = registry.with_resource(base_uri + schema_file_path.name, resource=resource)
+    return registry
+
+
+def validate_metaschemas(
+    reader: Reader,
+    collector: Collector = Collector.default,
+    types: Optional[TypeMapping] = None,
+    get_registry: Callable[[Reader, str], referencing.Registry] = _default_get_registry
```

Comment:
This was a slight diversion from the existing pattern.  

I wanted to make this unit testable without patching, and I wasn't sure about the best way to include the concept of "reading the metaschema files".  At first, I added it to the `Reader` class, but opted to implement it this way because it was specific to this validation function.